src/dotnet-core-uninstall/dotnet-core-uninstall.csproj

```diff
@@ -35,4 +36,13 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="PublishSingleFile" AfterTargets="Build">
```

Comment:
`AfterTargets="Build"` might not be very good. since you make build do actually publish

and use `MSBuild` task (run a new msbuild instance) is not necessary. You can make this task depends on `publish`.

Did you talk to Arcade team? I am not sure this is the right approach

Reply:
I talked to @livarcocc and someone in 1100. I think this is what they recommended.

Reply:
is the MSBuild task ok as well? i think it is will have an extra unnecessary msbuild instance

Reply:
We can play around with making this target depend on the publish target and see how that goes.

Reply:
FYI, making this target depend on the publish target does not work.